docs/Release_notes.md

```diff
@@ -1,6 +1,59 @@
 V2X-Hub Release Notes
 ---------------------------------
 
+Version 7.7.0, released Aug 14th, 2023
+--------------------------------------------------------
+
+**Summary:**
+V2X Hub release 7.7.0 introduces key enhancements and fixes to improve data streaming, upgrade MySQL to version 8.0, and support multi-architecture Docker deployments. This release includes updates to initialization scripts and addresses clock reference issues in simulation mode. Additionally, this release removes obsolete Docker tags and makes the SENSOR_JSON_FILE_PATH an optional environment variable to enhance flexibility in simulation setups.
+
+**<ins>V2X Hub CDASim Functionalities </ins>**
+
+Enhancement in this release:  
```

Comment:
Match the rest with how we presented to the client.

Reply:
updated